### ISSUE TITLE
Pagination issue

### ISSUE THREAD

**Issue by stalakerob** [Nemuritor01/css-swipe-card#3]
I'm using this card and like it very much. There is a little issue or my config is missing something. I have multiple Views each using this card. When I swipe to a card in one view and then go to another view and back to the one I swiped then it shows card 1 but pagination is still showing the card I swiped to. 

Example:
View 1 has 3 cards. I swipe to card 2.
Then I click on View 2 which also includes this component.
Then I click View 1 again. It shows card 1 but pagination is still on card 2.

**Comment by Nemuritor01** [comment]
Hi. Thanks for your feedback. Do you have this issue with the latest release v0.8 as well?
Ive had it with the release before and changed the code. Maybe also try to clear browser cache and on your phone the app cache.
if its not working, please let me know.

**Comment by stalakerob** [comment]
Yes, I am using v0.8. Clearing the caches did not help unfortunately.

**Comment by Nemuritor01** [comment]
Strange. Can you post your card config. I cant reproduce this failure.
So you use the same/similar card on two dashboard views. Do they have same cardId (just as information. It should even work with same cardId)

**Comment by stalakerob** [comment]
It is 1 dashboard with multiple views. Each of those views uses the css-swipe-card. And each swipe card has multiple cards (pages).
Each swipe card has a unique CardID. I realized that if I click another dashbord and go back then pagination is correct. Only if I switch between views of the same dashboard this issue occurs. I tested multiple browsers but they all behave the same.
[view_1.txt](https://github.com/user-attachments/files/16676555/view_1.txt)
[view_2.txt](https://github.com/user-attachments/files/16676556/view_2.txt)

**Comment by Nemuritor01** [comment]
Its hard to debug, without being able to reproduce the failure.
But I will have a look /  ich schaus mir an ;)

**Comment by stalakerob** [comment]
Danke sehr!
I created a test environment that is much simpler:
- a dashboard with 2 views. 
- the first one is using the attached code. 
[view.txt](https://github.com/user-attachments/files/16687755/view.txt)
- The second view is empty.
If I swipe on the first one and then click the second (empty) view and then go back to the first view it is the same issue as above.

**Comment by stalakerob** [comment]
Attached please also find 

https://github.com/user-attachments/assets/9736fe20-5d44-4288-a451-17311b21644b

screen recording of this dashboard.

**Comment by Nemuritor01** [comment]
Thanks for providing the information. Pretty strange. Thats working perfectly on my dashboard. I check the code and see, whatbI can do. This looks as if the card doesnt rerender.

**Comment by stalakerob** [comment]
I use Windows 11 and Android. Do you use Apple devices?

**Comment by Nemuritor01** [comment]
same, but my wife has an iphone, so I need to test with IOS as well.
